### ISSUE TITLE
KML.js bombs out on empty name "<name></name>"

### ISSUE THREAD

**Issue by mickfrancis** [shramov/leaflet-plugins#256]
There is an issue with empty names when using KML.js: If the KML file contains <name></name> the following code (around line 210 in my copy) bombs out:
```javascript
if (el.length) {
    name = el[0].childNodes[0].nodeValue;
}
```
Suggested fix:
```javascript
var ch;
if (el.length) {
    ch = el[0].childNodes;
    if (ch && ch.length) {
        name = ch[0].nodeValue;
    }
}
```

Kind regards,

Mick.

**Comment by brunob** [comment]
Have you tried with an up to date version of the script ? This seems fix since 49b6022c0ab33d56c1c79acca796220bc7ddb2d9

**Comment by mickfrancis** [comment]
Hi,

Ah – thanks for this!

Kind regards,

Mick Francis
Senior Software Engineer

direct:      +44 (0) 1344 350427
mobile:   +44 7981 027106
e-mail:    mick.francis@lightfootsolutions.com <mailto:mick.francis@lightfootsolutions.com%20>
web:         www.lightfootsolutions.com<http://www.lightfootsolutions.com>

Lightfoot Solutions and signalsfromnoise®(sfn) – Smarter Insights, Connected Thinking

[lightfoot_logo_blue]
Lightfoot Solutions UK Limited, Richmond House, Oldbury, Bracknell, Berkshire, RG12 8TQ.
Tel: +44 (0) 1344 350400. Fax:+44 (0) 1344 457550. Registered Office No. 06988557

Important Notice
This e-mail is confidential, may be legally privileged, and is for the intended recipient only. Access, disclosure, copying, distribution or reliance on any of it by anyone else is prohibited and may be a criminal offence. Please delete or destroy all copies immediately if obtained in error and e-mail confirmation to the sender. Thank you.
